manifests/init.pp

```diff
@@ -211,6 +211,7 @@
   Stdlib::Absolutepath                          $unbound_conf_d                  = "${confdir}/unbound.conf.d",
   Unbound::Hints_file                           $hints_file                      = "${confdir}/root.hints",
   Optional[String[1]]                           $hints_file_content              = undef,
+  Hash[String, Unbound::Rpz]                    $rpzs                            = {},
```

Comment:
Does it make sense to allow empty strings? Otherwise we could do:
```suggestion
  Hash[String[1], Unbound::Rpz]                 $rpzs                            = {},
```